### DISCUSSION TITLE
Timepicker with only time

### DISCUSSION THREAD

**Discussion by reshma-r-p** [t1m0n/air-datepicker#667] [General]
_I am using air date picker for my application. But for time picker it shows calendar and time picker. I need to select only time. The thing is in view only it shows calendar. But after selecting it shows only the time i selected. Is there anyway to show only the time selection instead of entire calendar view_

**Comment by danijelGombac** [comment]
You can try

timepicker: true,
onlyTimepicker: true

**Comment by reshma-r-p** [comment]
ok..thanks. Its worked

**Comment by sbiwaichoon** [comment]
how is this work? can show full code here?

**Comment by jpgateaux** [comment]
Here is an update for AIR-DATEPICKER 3+

<h3>CSS</h3>
<pre>	
           .air-datepicker.only-timepicker .air-datepicker--navigation {display: none;}
           .air-datepicker.only-timepicker .air-datepicker--content {display: none;}
           .air-datepicker.only-timepicker .air-datepicker--time {border-top: none;	}
</pre>
<h3>HTML</h3>
<pre>
           &lt;input type="text" id="input"  /&gt;
</pre>	 
<h3>JS</h3>
<pre>
	 new AirDatepicker('#input', {
         timepicker: true,
	 dateFormat: ' ',
         classes: 'only-timepicker'
});
</pre>
